cmd/describe/cluster/cmd.go

```diff
@@ -48,6 +48,8 @@ var Cmd = &cobra.Command{
 	Run: run,
 }
 
+const detailsLink = "https://cloud.redhat.com/openshift/details/"
```

Comment:
Any chance that this can be aware of the environment? Probably a `switch` block with `ocmConnection.URL()` with a fallback to production.